template/deploy/helm/[[product]]-operator/values.yaml.j2

```diff
@@ -0,0 +1,74 @@
+# =============
+# This file is automatically generated from the templates in stackabletech/operator-templating
+# DON'T MANUALLY EDIT THIS FILE
+# =============
+
+# Default values for {[ operator.product_string }]-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: docker.stackable.tech/stackable/{[ operator.product_string }]-operator
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
```

Comment:
This is really dangerous to specify in `values.yaml`, since it'll be kept during upgrades even if the chart itself changed.

Reply:
We should undefine this. In the template, we set it to "appVersion" of we leave `tag` empty.